packages/admin/admin/src/alert/Alert.tsx

```diff
@@ -0,0 +1,176 @@
+import { ArrowRight, Check, Close, Error, Info, Warning } from "@comet/admin-icons";
+import { Button, ButtonProps, IconButton, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+type Action = Partial<Omit<ButtonProps, "children">> & {
+    text: ButtonProps["children"];
+};
+
+export interface AlertProps {
+    severity?: "info" | "warning" | "error" | "success";
+    title?: React.ReactNode;
+    text: React.ReactNode;
+    hideCloseButton?: boolean;
+    onCloseClick?: () => void;
```

Comment:
I'd call this `onClose`, which would be more in line with other MUI components (e.g., Modal)